### DISCUSSION TITLE
Missing key in repo API response

### DISCUSSION THREAD

**Discussion by faultylee** [community/community#24331] [Enterprise]
<p>I’ve been pondering over why my repo API call to one org is missing these keys</p>
<pre><code class="lang-auto">allow_merge_commit
allow_squash_merge
allow_rebase_merge
delete_branch_on_merge
</code></pre>
<p>But it appears in another org. Both token have the same access to the corresponding org, both having enterprise licenses, under the same enterprise, but different SSO integration. Any idea what happened?</p>
<p>EDIT: FYI I’m using personal access token, and both the user are org admin</p>

**Comment by faultylee** [comment, marked as answer]
<p>I just figured out, the one with missing keys are called from org/repos, when I retrieve individual repo, I managed to get all the keys :slight_smile:</p>

**Comment by nguyengoi** [comment]
Hi @faultylee, I have the same problem
I use the rest API: GET /repos/:owner/:repo (with the personal fine-grained token)

`
curl \
     -H "Accept: application/vnd.github+json" \
     -H "Authorization: Bearer ${GITHUB_TOKEN}" \
        https://api.github.com/repos/:owner/:repo
 `
 
 
But I can't get these fields.
So, What API do you use?
